src/cli/command-package.ts

```diff
@@ -22,14 +23,27 @@ import {readJson} from '../manifest-converter';
 
 export default async function run(options: CliOptions) {
   // Ok, we're updating a package in a directory not under our control.
-  // We need to be sure it's safe. In a future PR let's check with git, but
-  // for now, we'll ask the user to pass in a --force flag.
-  if (!options.force) {
-    console.error(
-        `When running modulizer on an existing directory, ` +
-        `be sure that all changes are checked into source control. ` +
-        `Run with --force once you've verified.`);
-    process.exit(1);
+  // We need to be sure it's safe.
+  try {
+    await new Promise((resolve, reject) => {
+      exec('git status -s', {
+        cwd: options.in || process.cwd()
+      }, (error, stdout, stderr) => {
+        if (error || stderr || stdout) {
+          return reject();
```

Comment:
If an actual `error` happens it's probably not a git clean vs. unclean issue. Reject the error object and report it somehow below in the catch.

Reply:
The only `error` I was noticing was that the command wasn't found. We could either let this throw or do the `checkCommand` function and handle it. I'm not sure how you guys want to handle that case so I just left it as throwing for now.

Reply:
Sorry, I meant that I just meant to make sure a true error wasn't swallowed and reported under "hey your git repo is dirty". `reject(err)` and then doing something with the caught error would work.

The new change accomplishes this though too. If `exec()` rejects, the true error message will halt and bubble up.